packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -355,6 +357,10 @@ export class AuroraMysqlEngineVersion {
     return new AuroraMysqlEngineVersion(`5.7.${addStandardPrefix ? 'mysql_aurora.' : ''}${minorVersion}`);
   }
 
+  private static builtIn_8_0(minorVersion: string, addStandardPrefix: boolean = true): AuroraMysqlEngineVersion {
+    return new AuroraMysqlEngineVersion(`8.0.${addStandardPrefix ? 'mysql_aurora.' : ''}${minorVersion}`, '8.0');
+  }
```

Comment:
I doubt we will ever use this with the second parameter as `false`, so let's just remove it completely for now:

```suggestion
  private static builtIn_8_0(minorVersion: string): AuroraMysqlEngineVersion {
    return new AuroraMysqlEngineVersion(`8.0.mysql_aurora.${minorVersion}`, '8.0');
  }
```

This is a private method, we can always add back the parameter if we need to.